resources/home/dnanexus/generate_workbook.py

```
@@ -0,0 +1,97 @@
+
+        if file_type == "vcf":
+            data = vcf.open_vcf(file)
+        elif file_type == "xls" or file_type == "csv":
+            data = excel.open_file(file, file_type)
+        elif file_type == "html":
+            data = html.open_html(file)
+
+        inputs[name]["data"] = data
+
+    # get images and tables from the html file
+    html_images = html.get_images(inputs["supplementary_html"]["data"])
```

Ruff (0.8.2)

Local variable `html_images` is assigned to but never used

Remove assignment to unused variable `html_images`

(F841)